index.js

```diff
@@ -87,11 +87,19 @@ module.exports = function (schema, options) {
     var mainUpdateMethod = mongooseMajorVersion < 5 ? 'update' : 'updateMany';
     var mainUpdateWithDeletedMethod = mainUpdateMethod + 'WithDeleted';
 
-    function updateDocumentsByQuery(schema, conditions, updateQuery, callback) {
+    function updateDocumentsByQuery(schema, conditions, updateQuery, options, callback) {
```

Comment:
🤏 We should probably go with a different name to options since it's used in the module scope above.
```suggestion
    function updateDocumentsByQuery(schema, conditions, updateQuery, queryOptions, callback) {
```